pkgs/development/python-modules/pybibget/default.nix

```diff
@@ -0,0 +1,48 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, lxml
+, httpx
+, appdirs
+, aiolimiter
+, pybtex
+, pylatexenc
+, numpy
+, networkx
+, pytestCheckHook
+}:
+
+buildPythonPackage rec {
+  pname = "pybibget";
+  version = "0.1.0";
```

Comment:
Please add `pyproject = true` here

Reply:
Properly formatted commit messages and added pyproject as requested